pipelines/README.md

```diff
@@ -1,7 +1,7 @@
 # Temporary DECam Pipelines
 
-DECam-specific pipelines are temporarily stored here.
+DECam-specific pipelines jere are backwards-compatibility stubs only.
```

Comment:
Can we just remove the pipeline entirely and direct people to the correct place in the README?

(Also typo, `jere` -> `here`)

Reply:
I don't feel super strongly about this particular instance of backwards compatibility, but I hesitate to make structural changes without it, even if we don't formally have to wait a whole release cycle for removal after a deprecation.  I think I recall @parejkoj being concerned directly about compatibility in this change, and being relieved by this as a possible solution to it.